templates/uimodules/image.html

```diff
@@ -66,7 +66,7 @@ <h1>{{escape(sharedfile.get_title())}}</h1>
                             width="{{sized_width}}"
                             loading="lazy"
                             decoding="async"
-                            alt="{{escape(sharedfile.get_title())}}"></a>
+                            alt="{{escape(sharedfile.get_alt_text(raw=True) or 'No alt from user')}}"></a>
```

Comment:
Alt text of "No alt from user" is preferred over reusing the image title? This default sounds a little shame-y.

Reply:
Maybe "no alt provided" ? Like we'd really like users to provide something and titles often are not image descriptions, was I think the thinking there.

Reply:
Sure; but is it more useful to have something static, saying the effect of "Nothing" versus the title text which may describe the image, or may not.

Reply:
@bradchoate details here: https://github.com/MLTSHP/mltshp/issues/549#issuecomment-1609961684

Reply:
But agreed that "no alt provided" is better than "No alt from user"

Reply:
Yeah we do want to nudge users a little without them feeling set upon if they don't have the time or ability to provide it. It helps Search as well as low-vision or browsing-with-no-images users. It will gently change (or add) community expectations so we should think on the best way to do that.

Reply:
Pushed another commit with `No alt provided`. (I also pushed/reverted some files I didn't intend to, I can probably clean those up from the commit history if you'd prefer.)